src/js/edu-benefits/5490/config/form.js

```diff
@@ -125,13 +124,13 @@ const formConfig = {
           uiSchema: {
             'ui:title': 'Benefit selection',
             'view:benefitsDisclaimerChild': {
-              'ui:description': benefitsDisclaimerChild,
+              'ui:description': benefitsDisclaimer,
               'ui:options': {
                 hideIf: form => _.get('relationship', form) !== 'child'
               }
             },
             'view:benefitsDisclaimerSpouse': {
-              'ui:description': benefitsDisclaimerSpouse,
+              'ui:description': benefitsDisclaimer,
```

Comment:
Depending on @peggygannon's response below, view:benefitsDisclaimerChild and view:benefitsDisclaimerSpouse may need to be consolidated.